requestmanager/server.go

```diff
@@ -134,6 +137,16 @@ func (rm *RequestManager) requestTask(requestID graphsync.RequestID) executor.Re
 			Root:     cidlink.Link{Cid: ipr.request.Root()},
 			Selector: ipr.request.Selector(),
 			Visitor: func(tp traversal.Progress, node ipld.Node, tr traversal.VisitReason) error {
+				if lbn, ok := node.(datamodel.LargeBytesNode); ok {
```

Comment:
this seems like the relevant chunk for the partial traversal. looks good

Reply:
I think so right? in the meantime it fixes multiblock files as well.